ChangeLog.2010

```diff
@@ -8663,7 +8663,7 @@
 	  been renamed to gnc_resolve_file_path to be more consistent with
 	  the other functions. Lastly, this commit also adds a first
 	  implementation to work with a keyring to store and retrieve
-	  passwords, althoug
+	  passwords, although...
```

Comment:
This word is dangling

Reply:
See commit 6a6fe16. This very long commit message was cut off there.

So I think your suggestion is OK.

Reply:
Thanks 939d5630fcd2097dce1845ac476adee160d8c90b (dropped ??)